app/views/handlers/individual_response.py

```diff
@@ -300,39 +313,60 @@ def block_definition(self) -> Mapping:
                         self._list_name
                     ),
                 },
-                "description": [
-                    lazy_gettext(
-                        "For someone to complete a separate census, we need to send them an individual access code."
-                    ),
-                    lazy_gettext("Select how to send access code"),
-                ],
+                "description": self._build_question_description(
+                    has_postal_deadline_passed
+                ),
                 "answers": [
                     {
                         "type": "Radio",
                         "id": "individual-response-how-answer",
                         "mandatory": False,
                         "default": "Text message",
-                        "options": [
-                            {
-                                "label": lazy_gettext("Text message"),
-                                "value": "Text message",
-                                "description": lazy_gettext(
-                                    "We will need their mobile number for this"
-                                ),
-                            },
-                            {
-                                "label": lazy_gettext("Post"),
-                                "value": "Post",
-                                "description": lazy_gettext(
-                                    "We can only send this to an unnamed resident at the registered household address"
-                                ),
-                            },
-                        ],
+                        "options": self._build_handler_answer_options(
+                            has_postal_deadline_passed
+                        ),
                     }
                 ],
             },
         }
 
+    @staticmethod
+    def _build_handler_answer_options(has_postal_deadline_passed):
```

Comment:
Horses for courses, but it could be a cache property using self, not sure it would have any performance gains

Reply:
My point wasn't about performance as it will be negligible, it was from an OO point as the thing you are passing in is already a method of the parent class hence no need to be static. And that you could also cache it so ensure it won't be evaluated twice.